stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1005,6 +1008,8 @@ public enum OSVersion : CustomStringConvertible {
       return "watchOSSimulator"
     case Linux:
       return "Linux"
+    case FreeBSD:
+      return "FreeBSD"
```

Comment:
There is also `TestRunPredicate` below that should be updated, and then you should be able to write a test like this `validation-test/stdlib/StdlibUnittestLinux.swift` but for FreeBSD.